tools/RAiDER/cli/raider.py

```diff
@@ -501,6 +509,9 @@ def calcDelaysGUNW(iargs: list[str] = None) -> xr.Dataset:
 
     iargs = p.parse_args(iargs)
 
+    if not iargs.input_bucket_prefix:
+        iargs.input_bucket_prefix = iargs.bucket_prefix
+
```

Comment:
So hyp3 takes care of the creation of all the necessary prefixes.

Generally, we should never supply both `input_bucket_prefix` and `bucket_prefix` right?

Reply:
@cmarshak for any particular job, HyP3 will provide `bucket` as the HyP3 content bucket and `bucket_prefix` will be *that* job's ID. That's job's files will be uploaded to `s3://bucket/bucket-prefix`.

Becuase we ran RAiDER as a second step in an INSAR_ISCE job, the first step already uploaded a GUNW product to `s3://bucket/bucket-prefix`, and so RAiDER just looked there to download the GUNW.

Or in steps:
1. get GUNW from `s3://bucket/bucket-prefix`
2. add weather model correction to GUNW
3. upload modified GUNW to `s3://bucket/bucket-prefix` clobbering the old one. 

This PR makes it so we can do the above or do:

1. get GUNW from `s3://bucket/input-bucket-prefix`
2. add weather model correction to GUNW
3. upload modified GUNW to `s3://bucket/bucket-prefix`, which is empty


So:
* within HYP3_ISCE jobs where RAiDER is a step, We'll provide `--bucket` + `--bucket-prefix`
  https://github.com/ASFHyP3/hyp3/blob/develop/job_spec/INSAR_ISCE.yml#L138-L141
* as a standalone ARIA_RAIDER job, we'll provide `--bucket` + `--bucket-prefix` + `--input-bucket-prefix`
  https://github.com/ASFHyP3/hyp3/blob/raider-job/job_spec/ARIA_RAIDER.yml#L42-L49
*